### ISSUE TITLE
Overlayview position does not change when the position changes

### ISSUE THREAD

**Issue by T-Flynn** [googlemap-react/googlemap-react#32]
```
<OverlayView
        position={{ lng: poition.longitude, lat: poition.latitude }}
      >
```
How to change OverlayView's position?

**Comment by lucifer1004** [comment]
Fixed in `v0.1.7`.

**Comment by lucifer1004** [comment]
There is `console.log` call in `v0.1.7` and has been fixed in `v0.1.8`

**Comment by lucifer1004** [comment]
@all-contributors please add @T-Flynn for bug
